new_cert.tf

```diff
@@ -0,0 +1,53 @@
+variable "new_domains" {
+  type = list(string)
+  default = [
+    "cgr.dev",
+    "distroless.dev",
+    "images.wolfi.dev",
+  ]
+}
+
+// Enable Certificate Manager API.
+resource "google_project_service" "certmanager" {
+  service = "certificatemanager.googleapis.com"
+}
+
+resource "google_certificate_manager_dns_authorization" "this" {
+  for_each = toset(var.new_domains)
+  name     = replace("${each.key}", ".", "-")
+  domain   = each.key
+  labels   = {}
+}
+
+resource "google_certificate_manager_certificate" "cert" {
+  for_each = toset(var.new_domains)
+
+  name  = replace("${each.key}", ".", "-")
+  scope = "DEFAULT"
+
+  managed {
+    domains = [each.key]
+    dns_authorizations = [
+      google_certificate_manager_dns_authorization.this[each.key].id
```

Comment:
I might be mixing things up without a plan to look at, but I think you're missing the `*.` subdomain? as is this only builds certs for the tld (cgr.dev) and not (`*.cgr.dev`)

Reply:
gotcha, thanks for clarifying!